Makefile

```diff
@@ -1,3 +1,16 @@
+BASE_PATH = $(shell pwd)
+BIN_PATH := $(BASE_PATH)/bin
+
+export PATH := $(BIN_PATH):$(PATH)
+
+GOLANGCI_LINT := $(BIN_PATH)/golangci-lint
```

Comment:
Is this necssary for each binary? We should agree on these, so we would be consistent across repositories

Reply:
it's discussable, but I see 2 pros in such way:
-  you don't need to specify `bin` folder
- a full path to binary is used

Reply:
I personally prefer to not use such small variables for every binary that we use. It leads to variable hell and increases Makefile complexity. IMHO it's not a big deal to copy-paste `bin/something` in several places, but have really simple, easy-to-read makefile.

Reply:
Agree with @artemgavrilov

Reply:
as per discussion the variables removed